regression-test/suites/external_table_p0/paimon/test_paimon_catalog.groovy

```diff
@@ -72,10 +72,14 @@ suite("test_paimon_catalog", "p0,external,doris,external_docker,external_docker_
             def c15 = """select * from all_table where c15='a';"""
             def c16 = """select * from all_table where c16=true;"""
             def c18 = """select * from all_table where c18='2023-08-13 09:32:38.53';"""
-            def c19 = """select * from auto_bucket;"""
+            def c19 = """select * from auto_bucket order by user_id;"""
             def c20 = """select * from auto_bucket where dt="b";"""
             def c21 = """select * from auto_bucket where dt="b" and hh="c";"""
             def c22 = """select * from auto_bucket where dt="d";"""
+            def c23 = """select * from complex_tab order by c1;"""
+            def c24 = """select * from complex_tab where c1=1;"""
```

Comment:
24,25,26 is similar，keeping one is enough,can add some case  like 'SELECT m['a'] FROM simple_map;'

Reply:
Thanks for your suggestion, I have changed it